docs/src/content/en/reference/workflows/create-run.mdx

```diff
@@ -1,37 +1,16 @@
 ---
-title: "Reference: Workflow.createRunAsync() | Building Workflows | Mastra Docs"
-description: Documentation for the `.createRunAsync()` method in workflows, which creates a new workflow run instance.
+title: "Reference: Workflow.createRunAsync() | Workflows | Mastra Docs"
+description: Documentation for the `Workflow.createRunAsync()` method in workflows, which creates a new workflow run instance.
 ---
 
 # Workflow.createRunAsync()
 
-The `.createRunAsync()` method creates a new workflow run instance, allowing you to execute the workflow with specific input data.
+The `.createRunAsync()` method creates a new workflow run instance, allowing you to execute the workflow with specific input data. This is the current API that returns a `Run` instance. For the legacy `createRun()` method that returns an object with methods, see the [Legacy Workflows](../../legacyWorkflows/createRun.mdx) section.
 
-## Usage
+## Usage example
 
-```typescript
```

Comment:
We're losing information here (in the form of an example), so this could go to the bottom of the doc with the `## Example` heading